### ISSUE TITLE
Shadow priest without 4pc only recommends mind sear for 3+ targets

### ISSUE THREAD

**Issue by Jackpkmn** [Hekili/hekili#1572]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

The Shadow Priest rotation only recommends dots, then mind sear then things that can be cast during mind sear, and finally unholy nova (necrolord ability) for dpsing targets over 3. It won't recommend anything else and won't recommend popping cds when 3 or more targets are detected. If i disable all instances of mind sear it will get hung up on mind flaying instead, and if i disable all instances of mind flay it will get hung up on shadow word: pain then refuse to show any recommendations if i disable that. I tested both as Kyrian and as Necrolord the Kyrian version won't recommend boon of the ascended but the necrolord version will cast unholy nova. If it falls into the CD rotation before detecting 3 targets it will stay on track until power infusion falls off then go back to recommending only mind sear but will not enter the CD rotation if it detects 3 targets.

### How to Reproduce

Be shadow priest
get into any situation where hekili detects at least 3 targets while dots are rolling on the main target (including forcing aoe mode on the main rotation)
don't have 4pc tier set bonus (assumed, i can't actually test this myself)

### Snapshot (Link)

https://pastebin.com/U7WkmzTB

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/utFnJjiSnChS1HoAfaP2xN

### Additional Information

Importing a a simc rotation profile leads to the same issue, it gets hung up on mind searing against 3 targets or more which led me to think this might be a Hekili issue and not a profile issue. both the default and imported profile skip all the way down the priority list to the final mind sear entry as if all other entries have been disabled. If forced into aoe mode it will get hung up on the first mind sear entry that does not require you to have searing nightmare talented if you do not have that, and the second entry if you do talent it. I tested this by entering a caption for each instance and noting when each instance appears in the recommendation. Most of my testing has been done on the 5 target dummies in Elysian Hold but the same problem was exhibited in a M+ dungeon.

The snapshot provided is immediately following a profile reset with the only change being enabling cooldowns.

I cannot test with 4pc as i don't have it yet but this would still be an issue for not having 4pc on even if i did have it.

### Contact Information

Shippujinlai#5979

**Comment by Hekili** [comment]
I'm posting a potential fix, but first:

* Don't import a simc priority.  It helps nothing.  I import those and customize them for the addon.

I've added some changes to the Shadow priority for 1.0.7 that should address this issue.  It was related to Vampiric Touch and wanting to multidot but having "Recommend Target Swaps" turned off.

There's some other oddities to your snapshot, that could suggest other issues.  It looks like you might actually not be wearing gear (based on your haste), but you also don't have soulbinds or conduits.  That could be correct, though, if you're experimenting with Covenant swapping.

Give 1.0.7 a try and let me know how it goes.

**Comment by Jackpkmn** [comment]
Importing the Simc profile was a fix i tested becasue this worked for me in the past when the blood dk rotation was broken (like completely broken, suggesting double marrowrending on pull with another one after popping DRW even with CRW that's like 10 overcapped bones) I don't typically run the addon with an imported profile it was just something i tested.

I did try the recommend target swap option but it never actually recommended swapping targets.

As part of my testing i took off all my gear to make sure that none of my legendaries or trinkets were causing the problem. I typically play kyrian but i wanted to make sure that Kyrian itself wasn't causing the issue so swapped to necrolord which i don't play so don't have any conduits or soulbinds. 

I got 1.0.7 downloaded from curseforge i'll test it out soon as the servers come up.

edit: now that the servers are up i can 100% confirm it is fixed.
